### DISCUSSION TITLE
Addendum to blog post (actions => deprecate save state) to include the update about actions

### DISCUSSION THREAD

**Discussion by Zarthus** [community/community#62194] [Actions] [answered]
These two blog entries exist:

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
https://github.blog/changelog/2023-07-24-github-actions-update-on-save-state-and-set-output-commands/

In a CI Warning, you're linked to 

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Can we add an addendum to 2022-10-11's post to include the update?

![image](https://github.com/community/community/assets/3660408/3dae1af1-464b-438a-8475-1e75efd56a4c)

**Comment by rentziass** [comment, marked as answer]
Hey @Zarthus 👋 we just updated the [original post](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) to include a link to the new one, thank you very much for your suggestion! ❤️
